.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "scripts/lib/dok"]
```

Comment:
This adds a submodule that connects to my library, for displaying sprites using THREE.js